Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView_Tests.swift

```diff
@@ -50,21 +50,6 @@ class ChatChannelListItemView_Tests: XCTestCase {
         AssertSnapshot(view)
     }
         
-    func test_appearanceCustomization_usingAppearanceHook() {
-        class TestView: ChatChannelListItemView {}
-        TestView.defaultAppearance {
-            $0.subtitleLabel.font = .italicSystemFont(ofSize: 20)
-            $0.backgroundColor = .cyan
-        }
-        
-        let view = TestView().withoutAutoresizingMaskConstraints
-        
-        view.addSizeConstraints()
-        
-        view.content = content
-        AssertSnapshot(view)
-    }
-    
```

Comment:
We need to remove all snapshot files that have `_usingAppearanceHook` and are now obsolete 😄

Reply:
Yes :D Deleted :)